src/nova_router.erl

```diff
@@ -181,7 +181,8 @@ parse_url(Host, [{StatusCode, StaticFile}|Tl], Prefix, Value, Tree) when is_inte
     %% TODO! Fix status-code so it's being threated specially
     Tree0 = insert(Host, StatusCode, '_', Value0, Tree),
     parse_url(Host, Tl, Prefix, Value0, Tree0);
-parse_url(Host, [{StatusCode, {Module, Function}, Options}|Tl], Prefix, Value, Tree) when is_integer(StatusCode) ->
```

Comment:
Just looked through this commit and have a comment regarding this :). You don't need to transform the function-reference into a tuple.

Think this way you have proposed will be the new proper way of referencing a security-module and we will probably deprecate tuple-references in the future.

Reply:
i just pushed it because im parking this for a while while work is busy. the change in the nova_handler_value record grew complicated the evening i worked on it. so i was thinking this is a hack i would try out in a feat branch of our service based on nova so i can play with it.

i had some discussions with @Taure earlier about the xref issue and how it would be nice to have funs instead. i have daily talk with him, but we have never met, and i prefer to discuss by raising bad draft PRs than reporting issues  :)

what nova actually does internally with the routes i provide to it doesnt really matter to the app defining the routes and pointing out callbacks using funs. but given the efficiency guidelines indicate that applying a fun is slighly more efficient it is the right way to go internally.

https://www.erlang.org/doc/efficiency_guide/functions#function-calls

Reply:
It was not by any means critique on your solution - just that you could take a "shortcut" if you'd like not to transform your function-ref to a tuple :). Think that's the path we would like to take with Nova anyhow - try to skip the tuple-representation of MF.